src/markdown/markdown.cr

```diff
@@ -1,3 +1,17 @@
+# Markdown library parses Markdown text. It supports rendering to HTML text.
+# Note: This library is in its early stage. Many features and supports are still in development.
```

Comment:
Mmh, sorry but using "supports" as a noun in this context sounds weird to me, I associate that to physical structures that support holding something together or making it stronger.